config/locales/en.yml

```diff
@@ -164,11 +164,31 @@ en:
         eligibility_interface/teach_children_form:
           attributes:
             teach_children:
-              inclusion: Tell us if you have experience with teaching children
+              inclusion: Tell us whether you’re qualified to teach children aged between 5 and 16
         teacher_interface/name_and_date_of_birth_form:
           attributes:
             date_of_birth:
               inclusion: You must be 18 or over to use this service
+              blank: Enter your date of birth in the format 27 3 1980
+              less_than: Your date of birth must be in the past
+            given_names:
+              blank: Enter your given names
+            family_name:
+              blank: Enter your family name
+        teacher_interface/alternative_name_form:
+          attributes:
```

Comment:
Do we need an error message for the inclusion of `has_alternative_name`?

Reply:
Actually I think we don't to validate it at all. All its used for now is the conditional validation of the individual fields

Reply:
I think we still need to validate that the user at least choose one of the options, Yes or No.